crates/ra_assists/src/lib.rs

```diff
@@ -14,6 +14,7 @@ mod test_db;
 pub mod ast_transform;
 
 use hir::db::HirDatabase;
+use itertools::Either;
```

Comment:
```suggestion
use either::Either;
```

Reply:
Had to apply more changes than this one, so not applying it automatically.